backend/app/core/db/InitialDemoDataLoader.scala

```diff
@@ -165,6 +170,60 @@ class InitialDemoDataLoader @Inject() (
       .map(_ => (user1org, user2org, org))
   }
 
+  protected def initializePlaneConfig(projects: Seq[Project])(implicit
+      dbSession: DBSession,
+      userReference: UserReference): Future[PlaneConfig] = {
+    val apiKey = config.getString(
+      "lasius.plane.api_key"
+    )
+
+    if (apiKey == "none") {
+      logger.info("Skipping Plane configuration")
+      return Future.successful(null)
+    }
+    val planeAuth = PlaneAuth(
+      apiKey = apiKey,
+    )
+
+    val planeSettings = PlaneSettings(
+      checkFrequency = 60000
+    )
+
+    val projectConfigList = projects.map(p => {
+      val labels = p.key match {
+        case "KnowHow" => Set("Infrastructure")
+        case _         => Set.empty[String]
+      }
+
+      PlaneProjectMapping(
+        p.getReference().id,
+        PlaneProjectSettings(config.getString(
+                               "lasius.plane.project_id"
+                             ),
+                             maxResults = Some(100),
+                             None,
+                             Some(p.key),
+                             PlaneTagConfiguration(useLabels = true, labels))
+      )
+    })
+
+    val planeConfig = PlaneConfig(
+      PlaneConfigId(),
+      "organize.tegonal.com - Internal Stuff",
```

Comment:
I would either remove plane integration from demo data, make it configurable for other use cases as well (name) or create another tegonal specific data loader which we reference in our test instance.

Reply:
I removed it from demo data